bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBulkMutation.java

```diff
@@ -29,10 +28,13 @@
 public interface IBulkMutation {
 
   /**
-   * Send any outstanding {@link RowMutation} and wait until all requests are complete.
+   * Sends any outstanding {@link RowMutation} and wait until all requests are complete.
```

Comment:
This depends on who is doing the activity. If you use 'sends' here, it should be 'waits'. otherwise, if you are telling the user then, it  has to be "Send..." without any change.